### PR TITLE
engine: Use drop-in for logging configuration

### PR DESCRIPTION
**Pull request by tinez**
Requires: https://github.com/oVirt/ovirt-engine/pull/553

Signed-off-by: Marcin Sobczyk <msobczyk@redhat.com>

### CODE REVIEW

**Review by michalskrivanek — CHANGES_REQUESTED** [review]
Please keep it conditional, I like to run with default logging. 

I don't think we needed multiple vars, but ok...

File: ost_utils/ansible/collection.py [hunk 1]
```diff
@@ -79,6 +79,12 @@ def engine_setup(
     host_name = socket.gethostname()
     host_ip = socket.gethostbyname(host_name)
 
+    ansible_engine.copy(
+        content='ORG_OVIRT_LOG_LEVEL="DEBUG"\\nKEYCLOAK_LOG_LEVEL="DEBUG"\\nCORE_BLL_LOG_LEVEL="DEBUG"\\nROOT_LOG_LEVEL="DEBUG"',  # noqa: E501
+        dest='/etc/ovirt-engine/engine.conf.d/98-logging.conf',
```
Comment (michalskrivanek) [inline ost_utils/ansible/collection.py:84]: also, let's unite the two confs. ansible is horrible, not adding a new call is a plus.

Comment (tinez) [reply ost_utils/ansible/collection.py]: Ack

**Comment by tinez** [comment]
> Please keep it conditional, I like to run with default logging.

Done.

**Review by michalskrivanek — APPROVED** [review]
thanks

**Pull request merged**